projects/plugins/social/src/js/components/icon-text-section/styles.module.scss

```diff
@@ -0,0 +1,58 @@
+.column {
+	grid-column: 1 / -1;
+	display: grid;
+	grid-template-columns: auto 1fr;
+	grid-template-rows: repeat( 2, auto );
+	column-gap: calc( var( --spacing-base ) * 3 );
+	row-gap: calc( var( --spacing-base ) * 2 );
+
+	@supports ( display: grid ) {
+		> * + * {
+			margin-top: 0;
+		}
+	}
+
+	@media ( min-width: 600px ) {
```

Comment:
I think it would be good if we use [useBreakpointMatch](https://github.com/Automattic/jetpack/tree/trunk/projects/js-packages/components/components/layout/use-breakpoint-match) here as well (and in all the other media queries).

Reply:
Done!